pkg/minikube/cluster/cluster.go

```diff
@@ -140,11 +144,23 @@ func DeleteHost(api libmachine.API) error {
 
 // Un-register host VM
 func unregister(host *host.Host) error {
+	if !minishiftConfig.InstanceConfig.Register {
+		return nil
```

Comment:
@LalatenduMohanty Which error? This is when a system is already unregister why we want to send a error here?